### PR TITLE
Bump version to 0.6.16-ens.13

### PR DESCRIPTION
**Pull request by jmasseo**

### CODE REVIEW

File: terraform.rb [hunk 1]
```diff
@@ -52,7 +52,7 @@ def install
     end
 
     cd terrapath do
-      # v0.6.12 - source contains tests which fail if these environment variables are set locally.
+      # v0.6.13 - source contains tests which fail if these environment variables are set locally.
```
Comment (josephholsten) [inline terraform.rb]: `v0.6.16`?

Comment (mlrobinson) [reply terraform.rb]: Comments with version numbers, I think it should stay the same, as that should denote when this error started (unless we've fixed it).

Comment (jmasseo) [reply terraform.rb]: I think i just searched for 12 and changed to 13.

**Review by mlrobinson — APPROVED** [review]

**Review by josephholsten — APPROVED** [review]
¡delightfully boring!

**Pull request merged**